fixtures/showtec/performer-2500.json

```
@@ -0,0 +1,225 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Performer 2500",
+  "categories": ["Dimmer", "Color Changer"],
+  "meta": {
+    "authors": ["purple", "Luke Nelson"],
+    "createDate": "2022-10-18",
+    "lastModifyDate": "2022-10-18"
+  },
+  "links": {
+    "productPage": [
+      "https://www.highlite.com/en/33141-performer-2500-fresnel.html"
+    ],
+    "manual": [
+      "https://all-guidesbox.com/manual/2217476/showtec-performer-2500-fresnel-tungsten-operation-user-s-manual-40.html"
+    ],
+    "video": ["https://vimeo.com/439925462"]
+  },
+  "physical": {
+    "dimensions": [450, 380, 470],
+    "weight": 10,
+    "power": 265,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "LED",
+      "lumens": 6959
+    },
+    "lens": {
+      "name": "Fresnel",
+      "degreesMinMax": [10, 60]
+    }
+  },
+  "availableChannels": {
+    "Master Dimmer": {
+      "fineChannelAliases": ["Master Dimmer fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Warm White": {
+      "fineChannelAliases": ["Warm White fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Warm White"
+      }
+    },
+    "Natural White": {
+      "fineChannelAliases": ["Natural White fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "CCT": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 35],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "2700 K"
```

Here, the whole DMX range results in the same color temperature, thus there is no need for the `Start`/`End` properties:

```suggestion
          "colorTemperature": "2700 K"
```